test/services/svc-service.test.ts

```diff
@@ -0,0 +1,84 @@
+import { expect, test } from '@oclif/test'
+import { CLIError } from '@oclif/errors'
+import * as fs from 'node:fs'
+import * as path from 'node:path'
+import { TempDirHelper } from '../helpers/temp-dir-helper'
+import { SvcService } from '../../src/services/svc-service'
+import { BundleDescriptorService } from '../../src/services/bundle-descriptor-service'
+
+describe('svc-service', () => {
+  let bundleDirectory: string
+  let bundleDescriptorService: BundleDescriptorService
+  const tempDirHelper = new TempDirHelper(__filename)
+
+  before(() => {
+    bundleDirectory = tempDirHelper.createInitializedBundleDir('sample-bundle')
+    fs.rmSync(path.resolve(bundleDirectory, 'svc', 'mysql.yml'))
+    bundleDescriptorService = new BundleDescriptorService(bundleDirectory)
+  })
+
+  afterEach(() => {
+    const bundleDescriptor = bundleDescriptorService.getBundleDescriptor()
+    bundleDescriptorService.writeBundleDescriptor({
+      ...bundleDescriptor,
+      svc: []
+    })
+  })
+
+  after(() => {
+    fs.rmSync(path.resolve(tempDirHelper.tmpDir), {
```

Comment:
This is not needed, since it is already done by `TempDirHelper`

Reply:
done